src/Message/Request/FetchIssuersRequest.php

```diff
@@ -6,7 +6,7 @@
 
 /**
  * Class FetchIssuersRequest
- * @package Omnipay\Paynl\Message\Request
+ * @package Omnipay\PaynlV3\Message\Request
```

Comment:
@Tomgroot did they just forget to adjust this? This is still from the fork right?

Reply:
namespace has been changed to `Omnipay\PaynlV3` by them, do we want to change this back to the old one?

Reply:
No it's fine, was just wondering what kind of job they did